src/components/YoutubePlayer.jsx

```diff
@@ -0,0 +1,11 @@
+import ReactPlayer from 'react-player'
+
+const YoutubePlayer = ({ videoKey }) => (<ReactPlayer 
```

Comment:
- To follow the Single Responsibility Principle (SRP), the URL construction logic should be separated into its own function.
-
- You could add a helper function to build the video URL.
- Keep the YoutubePlayer component as a container that manages these elements.